task8/ServletsOnMaven/src/main/java/ServletsOnPosts/GetSizeServlet.java

```diff
@@ -0,0 +1,60 @@
+package ServletsOnPosts;
+
+import Posts.UtilClasses.DateInterval;
+import Posts.UtilClasses.FilterParams;
+import Posts.UtilClasses.JSONDecorator;
+import WorkWithDB.WorkWithDB;
+
+import javax.annotation.Resource;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class GetSizeServlet extends HttpServlet {
+
+    @Resource(name="jdbc/mydb")
+    private DataSource dataSource;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String authorParam = req.getParameter("author");
+        String leftDateParam = req.getParameter("left");
+        String rightDateParam = req.getParameter("right");
+        String hashTagsParam = req.getParameter("hashTags");
+        List<String> hashTags = hashTagsParam == null ? new ArrayList<>()
+                : new ArrayList<>(Arrays.asList(hashTagsParam.split("[\\s\\n\\t\\r]")));
```

Comment:
Better add `+` after `[]` in regexp, so that multiple dividers standing side-by-side handled